pkg/api/dashboard_test.go

```diff
@@ -88,9 +88,28 @@ func TestGetHomeDashboard(t *testing.T) {
 		name                  string
 		defaultSetting        string
 		expectedDashboardPath string
+		req                   *contextmodel.ReqContext
 	}{
-		{name: "using default config", defaultSetting: "", expectedDashboardPath: "../../public/dashboards/home.json"},
-		{name: "custom path", defaultSetting: "../../public/dashboards/default.json", expectedDashboardPath: "../../public/dashboards/default.json"},
+		{
+			name:                  "using default config",
+			defaultSetting:        "",
+			expectedDashboardPath: "../../public/dashboards/home.json",
+			req:                   &contextmodel.ReqContext{SignedInUser: &user.SignedInUser{}, Context: &web.Context{Req: httpReq}},
+		},
+		{
+			name:                  "custom path",
```

Comment:
Could you maybe clarify what these test do and how they defer?

Reply:
Description updated